README.md

```diff
@@ -56,6 +56,18 @@ With Level 0 data, this can lead to a very large netCDF file.
 >>> rpg2nc('/path/to/files/*.LV0', 'huge_file.nc')
 ```
 
+### Converting multiple files to NetCDF4
+It is possible to convert multiple lv0 or lv1 files with the following function.
+Every file with extension .LV0, .lv0, .LV1 or .lv1 in every subdirectory of the specified path will be included in the convertion.  
+Optionally, the user can exclude level 0 files by switching to `False` the argument `include_lv0`; by default it will include them.
+```python
+>>> from rpgpy import rpg2nc_multi
+>>> rpg2nc_multi('/path/to/myfiles', include_lv0=True, base_name"foo")
+```
+If no path is made explicit, the function will by default take as argument the current directory
+and write the converted file in it.  
+If the parameter `base_name` is specified, the function will rename the new files as `basename_oldname`.
+
```

Comment:
Just wondering, could we make this simpler using the existing `rpg2nc` function with a new optional argument like `concatenate = False` (where default would be `True`)?

Reply:
We could do that. However, it would imply a subtle change of the semantics. What I mean is that the user, when using the function `rpg2nc`, would need to specify the path as the file(s)' path, while for the function `rpg2nc_multi` the user will need to explicit the directory path.
I guess it could lead to some confusion. What do you think?

Reply:
Hey @tukiains , did you have the chance to read my comment above?

Reply:
Hmm, you mean that it would not be obvious where the output files would be written? I guess it would have to be something like this:
`rpg2nc('/path/to/files/*.lv0', output='/foo/bar/file.nc')` and
`rpg2nc('/path/to/files/*.lv0', output='/foo/bar/', concatenate=False)` with some default functionality if `output` is not defined. Does it get too complicated? The two functions are almost the same and I wish to avoid duplicate code if possible but If you think it only complicates matters we can keep these separate (at least for now).

Reply:
Hi Simo, what I meant was not regarding the output parameter but the input files: at the moment, the function `rpg2nc_multi` takes as argument a **directory** (not a list of files) and walk in every subdirectory to find LV0 and LV1 files, then by default it writes the new files in the same directory which was passed as argument. I thought that by designing it this way, the user wouldn't have to bother to move files before any conversion - if those are not in the same folder already.
So I guess the two solutions are different. Nevertheless, I could think a way to reach the function you wrote in the comment above! Just let me know

Reply:
Okay, let's keep these separated at least for the time being.